server/data/data-jmap/src/test/scala/org/apache/james/jmap/api/upload/UploadServiceContract.scala

```diff
@@ -108,6 +108,7 @@ trait UploadServiceContract {
         .block())
 
     // Exceed 100 bytes limit
+    Thread.sleep(500)
```

Comment:
Q: any chance we can use await?

Reply:
It is a difference.
Here I want the `cleanupUploadIfNeeded` completed (it was called 10 times) before triggering "exceed".